odxtools/filter.py

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from typing import Any
+from xml.etree import ElementTree
+
+from .exceptions import odxrequire
+from .odxdoccontext import OdxDocContext
+from .odxlink import OdxLinkDatabase, OdxLinkId
+from .snrefcontext import SnRefContext
+from .utils import read_hex_binary
+
+
+@dataclass(kw_only=True)
+class Filter:
+    filter_start: int
```

Comment:
a class for just an int?

Reply:
it looks like it. but FILTER is a base class like PARAMETER, i.e., all instances of FILTER are either `SIZEDEF-FILTER` or `ADDRDEF-FILTER` (which need to be specified via `xsi:type`)...

Reply:
So abstract class?

Reply:
yes. That said, we cannot make it an ABC on the python side because this would break the `.from_et()` mechanism because this requires the base class to be instantiable...